test/smoke/src/areas/positron/top-action-bar/top-action-bar.test.ts

```diff
@@ -0,0 +1,108 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (C) 2024 Posit Software, PBC. All rights reserved.
+ *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { join } from 'path';
+import { expect } from '@playwright/test';
+import { Application, Logger } from '../../../../../automation';
+import { installAllHandlers } from '../../../utils';
+
+/*
+ * Top Action Bar test cases
+ */
+export function setup(logger: Logger) {
+	describe.only('Top Action Bar', () => {
+		// Shared before/after handling
```

Comment:
Accidentally checked in with .only(

Reply:
oops 😓 I may need to add a pre-commit hook at some point to help avoid this!